readme.md

```diff
@@ -1,10 +1,10 @@
 # Hello world
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+- [DEMO LINK](https://vetalek.github.io/layout_hello-world/)
+- [TEST REPORT LINK](https://vetalek.github.io/layout_hello-world/report/html_report/)
```

Comment:
No ,just updating the links won't help, you need to run `npm run deploy` once more